### ISSUE TITLE
Outdated App

### ISSUE THREAD

**Issue by a5464c9f32** [wosk/flipperzero-tpms#22]
when i install the compiled file on my flipper, i get an error about the app being not up to date. How can i recompile the app for the latest Flipper versions?

**Comment by jessenolan1** [comment]
Not sure if this repo is maintained anymore but i forked it and just built the app again. I created a release on [my fork](https://github.com/jessenolan1/flipperzero-tpms) if you wanna check it out.

**Comment by a5464c9f32** [comment]
Thank you! I'll check it out.

**Comment by Mureren4000** [comment]
Jessenolan1
Your version 0.21, works fine, but my Flipper say it's still old version, and if I open app and see About, it's still v.0.1 :)

**Comment by MasterMuggs** [comment]
I just. updated a copy by forking as well should work with flipper 1.2.0 [https://github.com/MasterMuggs/flipperzero-tpms](url)

**Comment by cocus** [comment]
> I just. updated a copy by forking as well should work with flipper 1.2.0 [https://github.com/MasterMuggs/flipperzero-tpms](url)

Thanks, but is it possible that it the -> button doesn't work? I can't trigger the "re-learn" menu.

EDIT: Pressing the back button shows the menu to re-learn, not sure if it's actually transmitting the 125kHz stuff or not, I should try it!

**Comment by MasterMuggs** [comment]
Yeah I was unable to get it to work as well. I am not sure if that is something the original maker can help out with.

**Comment by cocus** [comment]
> Yeah I was unable to get it to work as well. I am not sure if that is something the original maker can help out with.

I've put a scope probe near the device, selected the RELEARN option to be ON, then pressed the RIGHT arrow. A 170kHz (more or less) sine appears. I think it's okay, but the frequency might be off. I don't know if it's my setup or something changed in the API from the OS itself.

EDIT: No, it was my setup. It's 125kHz, no problem with that. Just no UI feedback that it's actually transmitting it :)

**Comment by jhonnybonny** [comment]
https://github.com/jhonnybonny/tpms_reader_flipper_zero
